### PR TITLE
Eng 9516 a default query timeout for reads 

### PR DESCRIPTION
**Pull request by JadenW**
so that I don't think my new VoltDB cluster is hung

With default value of 10 Seconds.
Also set this default value to SQLCommand

### CODE REVIEW

File: src/frontend/org/voltdb/utils/SQLCommand.java [hunk 1]
```diff
@@ -1028,7 +1028,7 @@ private static void printUsage(int exitCode)
         + "  and returns to the command prompt. Default: true.\n"
         + "\n"
         + "[--query-timeout=millisecond_number]\n"
-        + "  Read-only queries that take longer than this number of milliseconds will abort. Default: no timeout.\n"
+        + "  Read-only queries that take longer than this number of milliseconds will abort. Default: 10 Seconds.\n"
```
Comment (akhanzode) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: Use constant here as well in generating message.

**Comment by akhanzode** [comment]
When query times out do we give in error message what user can do to change this?

File: src/frontend/org/voltdb/ProcedureRunner.java [hunk 2]
```diff
@@ -1139,6 +1139,18 @@ else if (e.getClass() == SpecifiedException.class) {
        // ensure the message is returned if we're not going to hit the verbose condition below
        if (expected_failure || hideStackTrace) {
            msg.append("  ").append(e.getMessage());
+           if (e instanceof org.voltdb.exceptions.InterruptException) {
```
Comment (vtkstef) [inline src/frontend/org/voltdb/ProcedureRunner.java]: if you have to have the timeout value reflected in VoltDB.Configuration and get it with `VoltDB.instance().getConfig()` this bit i adding two map lookups for every transaction, which will simply not do.

Comment (akhanzode) [inline src/frontend/org/voltdb/ProcedureRunner.java]: Also check m_isReadOnly this message is only for RO queries.

**Pull request merged**